realm/realm-library/src/androidTest/java/io/realm/RealmConfigurationTests.java

```diff
@@ -91,15 +92,8 @@ public void tearDown() throws Exception {
         }
     }
 
-    private void clearDefaultConfiguration() throws NoSuchFieldException, IllegalAccessException {
-        final Field field = Realm.class.getDeclaredField("defaultConfiguration");
-        field.setAccessible(true);
-        field.set(null, null);
-    }
-
     @Test
-    public void setDefaultConfiguration_nullThrows() throws NoSuchFieldException, IllegalAccessException {
-        clearDefaultConfiguration();
+    public void setDefaultConfiguration_nullThrows() {
```

Comment:
I think that there is no need to clear default configuration in this test. Correct?